docs/.vuepress/sidebar.js

```diff
@@ -53,18 +53,22 @@ module.exports = {
       collapsable: false,
       children: [
         'guide/conditional-fields.md',
-        [ 'guide/async-components.md', 'Async components 🆕' ]
+        [ 'guide/async-components.md', 'Async components 🆕' ],
+        [ 'guide/hosting.md', 'Hosting in production 🆕' ]
       ]
     },
+    [ 'guide/upgrading.md', 'Coming from 2.x 🆕' ],
+    [ 'guide/contribution.md', 'Contribution guide 🆕' ]
+  ],
+  '/cookbook': [
     {
       title: 'Hosting',
       collapsable: false,
+      sidebarDepth: 2,
       children: [
-        [ 'guide/hosting.md', 'Hosting in production 🆕' ]
+        [ 'cookbook/ubuntu-setup.md', 'Ubuntu hosting setup 🆕' ]
```

Comment:
This should be `ubuntu-hosting-setup` since we probably have separate guidance (or we will) on ubuntu as a dev environment.